.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  build:
```

Comment:
Please define an orb and execute the tests for the different databases in parallel.

Reply:
ok